src/styles/main.scss

```
@@ -1,3 +1,224 @@
-body {
+
+@mixin display-positional ($general, $horisont, $vertical) {
+  display: $general;
+  justify-content: $horisont;
+  align-items: $vertical;
+}
+
+@mixin text-style-fw-fs-lh ($weight, $size, $height, $color) {
+  font-weight: $weight;
+  font-size: $size;
+  line-height: $height;
+  color: $color;
+}
+
+@mixin area-styles-w-h-brdr-brad-bg (
+  $width,
+  $height,
+  $border,
+  $radius,
+  $bg-color
+) {
+  width: $width;
+  height: $height;
+  border: $border;
+  border-radius: $radius;
+  background: $bg-color;
+}
+
+@mixin stars ($url, $margin-l) {
+  width: 16px;
+  height: 16px;
+  background-image: $url;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: $margin-l;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+h2,
+p,
+ul {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+.header {
+  @include display-positional(flex, space-between, center);
+  padding-left: 50px;
+  padding-right: 50px;
+  @include area-styles-w-h-brdr-brad-bg(100%, 60px, none, 0, #fff);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  @include display-positional(flex, center, center);
+
+  &__links {
+    @include display-positional(flex, center, center);
+
+    &-behavior {
+      @include display-positional(flex, center, center);
+
+      &:not(:last-child) {
+        margin-right: 20px;
+      }
+
+    }
+
+  }
+
+  &__link {
+    @include text-style-fw-fs-lh(500, 12px, 60px, #000);
+    text-decoration: none;
+    position: relative;
+  }
+  
+  &__link:hover {
+    color: #00acdc;
+  }
+
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active:after {
+  position: absolute;
+  content: "";
+  bottom: 0;
+  left: 0;
+  @include area-styles-w-h-brdr-brad-bg(100%, 4px, none, 2px, #00acdc);
+}
+
+.catalog {
+  padding: 50px 40px;
+
+  &__title {
+    position: absolute;
+    visibility: hidden;
+  }
+}
+
+@media (min-width: 1200px) {
+  .catalog {
+    padding: 50px 128px 128px;
+  }
```

Якщо додав це media в catalog, то можна записати його спрощенно. 
```suggestion
@media (min-width: 1200px) {
    padding: 50px 128px 128px;
  }
```